base-application.gradle

```diff
@@ -9,6 +9,7 @@ android {
     defaultConfig {
         minSdkVersion minVersion
         targetSdkVersion compileVersion
+        missingDimensionStrategy 'extraAPIs', 'defaults'
```

Comment:
This makes it so `integration-tests`, `api-tester` and `examples:purchase-tester` don't need flavors, and just use the `defaults` flavor.